systemd/nexus-init.sh

```diff
@@ -58,10 +78,9 @@ if ! podman inspect "$NEXUS_CONTAINER_NAME" ; then
         --conmon-pidfile "$NEXUS_PIDFILE" \
         --cidfile "$NEXUS_CIDFILE" \
         --cgroups=no-conmon \
-        -d \
```

Comment:
was this argument deprecated, or do we no longer run this as detached?

Reply:
Yes, deprecated.  The same change is in the basecamp init file